### DISCUSSION TITLE
My site not working

### DISCUSSION THREAD

**Discussion by anuradhadesigner** [community/community#22480] [Other Feature Feedback, Questions, & Ideas]
<p>when i make my site live like using <a href="https://github.com/anuradhadesigner.io/Netflix" rel="noopener nofollow ugc">https://github.com/anuradhadesigner.io/Netflix</a> it throughs error like deceptive site plz resove this issue as soon as possible because my all projects not working same issue arrives.</p>

**Comment by Katsute** [comment, marked as answer]
<p>The deceptive site warning is issued by the browser (Google Chrome), not GitHub. If you use Microsoft Edge the site loads without the warning. Ideally you should not be posting a site that looks exactly like another commonly used site, since that is what phishing sites do.</p>

**Comment by jfullstackdev** [comment]
<p>why do you think there is the error ?</p>
<p>at the Developers Tools, take a look at the error</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/0/4033c12e3f9f5d4f5751a56ce22728cc8c8235df.png" data-download-href="/uploads/short-url/99Xrou2XlNERuohzayA0iT6Rldd.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071363-0af71339-f11b-4954-b87a-898e29ccaaae.png" alt="image" data-base62-sha1="99Xrou2XlNERuohzayA0iT6Rldd" width="608" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/4033c12e3f9f5d4f5751a56ce22728cc8c8235df_2_608x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/4033c12e3f9f5d4f5751a56ce22728cc8c8235df_2_912x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/4033c12e3f9f5d4f5751a56ce22728cc8c8235df_2_1216x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/4033c12e3f9f5d4f5751a56ce22728cc8c8235df_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1265×1040 69.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/anuradhadesigner">@anuradhadesigner</a>,</p>
<p>Like <span class="mention">@katstute</span> pointed it out, this is not a GitHub problem but really your browser protecting you.</p>
<p>A lot of phishing attacks starts with a site that looks exactly like another one. If anything, that tells you you passed your CSS exercise here :slight_smile:.</p>
